pkg/data/examples.go

```diff
@@ -2333,3 +2335,33 @@ var ExampleAppWithGroups = registerExample(&Example{
 		},
 	},
 })
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////
+
+var ExampleAppWithGroups2 = registerExample(createExampleAppWithGroups2())
+
```

Comment:
seems that this example is not tested...

Reply:
added to analyzer tests